scripts/update-internal-dependencies.js

```diff
@@ -0,0 +1,85 @@
+const fs = require("fs").promises;
+
+const semver = require("semver");
+
+// get cli arguments. Any cli argument is assumbed to be the name of a package
+// dependency to update in other packages. If no extra cli arguments are
+// provided, then all packages will be updated.
+//
+// For example:
+//   `yarn update-internal-dependencies`
+//      updates everything
+//   `yarn update-internal-dependencies core-utils base-map`
+//      updates packages that depend on either @opentripplanner/core-utils or
+//      @opentripplanner/base-map
+const filteredPackages = process.argv.slice(2);
+if (filteredPackages.length === 0) {
+  console.log("Updating all internal @opentripplanner dependencies!");
```

Comment:
Is this a bad thing? The exclamation mark makes me think it's not something we should be doing. I think @binh-dam-ibigroup and my uncertainty about how this works speaks to the need to document how this should be used in the README.

Reply:
Removed scary exclamation point in 5163a219c2ab5072d9338041435293fda8877c42